src/shared/firewall-util.h

```diff
@@ -30,3 +31,33 @@ int fw_add_local_dnat(
                 const union in_addr_union *remote,
                 uint16_t remote_port,
                 const union in_addr_union *previous_remote);
+
+struct NFTSetContext {
+        int nfproto;
+        char *table;
+        char *set;
+};
+typedef struct NFTSetContext NFTSetContext;
```

Comment:
Not a matter, but we usually merge typedef and declaration of struct.

Reply:
merged